<!DOCTYPE html>
<html>
	<head>
		<meta charset="UTF-8">
		<title></title>
	</head>
	<body>
	</body>
	<script type="text/javascript">
		let pro=new Promise(function(resolve,reject){
             if(1<0){
             	resolve('1>0是对的')
             }else{
             	reject('1小于0是错误的')
             }
		})
		 console.log(pro)
//		  当异步操作失败后Promise对象返回的状态是'rejected',此时浏览器会抛出一个错误，错误内容就是reject()中的值

	</script>
</html>
